.gitignore

```diff
@@ -3,4 +3,7 @@ docker-compose.override.yml
 data/*
 !data/.gitkeep
 containers/custom/*
-!containers/custom/.gitkeep
\ No newline at end of file
+!containers/custom/.gitkeep
+.DS_Store
+daemon.log
+daemon.pid
```

Comment:
What creates these files? Do they need to be ignored?

Reply:
Correct me if I'm wrong - still really new to a lot of this dev ops-y stuff -  but this should be a generated file for when a daemon is running to store it's ID so it can't be run twice.

This must be created by either docker sync or unison.

I added to gitignore simply because it's a generated file. Not sure if this is correct or not, though.